cypress_test/integration_tests/common/helper.js

```diff
@@ -368,16 +368,16 @@ function checkIfDocIsLoaded(isMultiUser) {
 
 	if (isMultiUser) {
 		cy.cSetActiveFrame('#iframe1');
-		cy.getFrame().its('body').should('not.be.undefined');
+		cy.cframe('#iframe1').its('body').should('not.be.undefined');
```

Comment:
I guess we don't need to send the frameID since it is set at the line above? Same for line 375. If you agree then we can remove these with your next PR.